rcl/test/rcl/test_init.cpp

```diff
@@ -298,8 +298,30 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_options_acce
   const rcl_allocator_t * options_allocator = rcl_init_options_get_allocator(&init_options);
   EXPECT_TRUE(rcutils_allocator_is_valid(options_allocator));
 
+  size_t domain_id;
+  ret = rcl_init_options_get_domain_id(NULL, &domain_id);
+  ASSERT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+  ret = rcl_init_options_get_domain_id(&init_options, NULL);
```

Comment:
can you add `rcl_reset_error()` calls in the middle?
This will avoid messages saying that the global error message state was overridden

Reply:
okay, it it better to call.